pinot-common/src/test/resources/pql_queries.list

```diff
@@ -812,3 +812,8 @@ SELECT column1, column5, column6 FROM testTable ORDER BY column5, column9 WHERE
 select $docId, $segmentName, $hostName from mytable where $docId < 5 limit 50
 select $docId, $segmentName, $hostName from mytable where $docId = 5 limit 50
 select $docId, $segmentName, $hostName from mytable where $docId > 19998 limit 50
+SELECT SUM(met) FROM mytable WHERE timeConvert(DaysSinceEpoch,'DAYS','SECONDS') = 1394323200
```

Comment:
Map UDF is not yet merged

Reply:
it's ok, this is just for compile, not evaluate.